R/utilities.r

```diff
@@ -430,3 +430,24 @@ is_column_vec <- function(x) {
   dims <- dim(x)
   length(dims) == 2L && dims[[2]] == 1L
 }
+
+#' Parse a vector of expressions without silently dropping any items.
```

Comment:
Since this is not exported, I don't think it needs to be documented. A comment mentioning that `parse(text = text)` has subtle problems with link to issue should be adequate.

Reply:
I have a feeling future developers will appreciate the hints in the documentation. I know I was definitely surprised by the default behavior of `parse()` dropping things. Totally unexpected for me.

Regarding the link, is `#2864` sufficient or do you prefer to put a full url like `https://github.com/tidyverse/ggplot2/issues/2864`?

Reply:
Here, I'd actually prefer the full url since that is clickable in RStudio. I'd suggest something like:

```R
# Parse takes n lines and returns m _expressions_
# See https://github.com/tidyverse/ggplot2/issues/2864 for discussion
```